homeassistant/components/airvisual_pro/config_flow.py

```diff
@@ -52,10 +63,13 @@ async def async_validate_credentials(ip_address: str, password: str) -> dict[str
     except Exception as err:  # pylint: disable=broad-except
         LOGGER.exception("Unknown error while connecting to %s: %s", ip_address, err)
         errors["base"] = "unknown"
+    else:
+        measurements = await node.async_get_latest_measurements()
```

Comment:
Can this raise any exceptions?

Reply:
Good call: it can raise the same exceptions as `async_connect`.